ruby_sinatra/bin/setup

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 set -e
 . $(dirname "$0")/../../bin/silently
-gem list -i sinatra >/dev/null || silently gem install sinatra --no-ri
+gem list -i bundler >/dev/null || silently gem install bundler
+bundle check >/dev/null || bundle install --path tmp/bundle
```

Comment:
do a `silently bundle install...` here so that it only is chatty if something fails. if you look at the test output its super chatty